packages/textlint-fixer-formatter/test/textlint-fixer-formatter-test.js

```diff
@@ -0,0 +1,19 @@
+// LICENSE : MIT
```

Comment:
Can you convert it to TypeScript?

It is good that adding new test file as TypeScript.

Reply:
It makes sense. Thanks. I also found that only `assert` is not meaningful. Trying to fix.